jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -301,6 +219,17 @@ With the following, nested under `steps:` and customizing for your application a
 - See the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/#deploy) document for valid `deploy` options to configure deployments on CircleCI 2.0
 - Please read the [Deploy]({{ site.baseurl }}/2.0/deployment-integrations/) document for examples of deployment for CircleCI 2.0.
 
+## Environment Variables
+
+In CircleCI 2.0, all defined values are treated literally.
+When defining configuration variables like `working_directory` or `images`,
```

Comment:
I feel like this sentence could be deleted (it leads with the limitation but doesn't give enough details here to make it clear, so I'd rather just leave those to the env vars page).

Reply:
🙏 for letting me delete things @michelle-luna